sys-apps/util-linux/patches/util_linux-2.32.patchset

```diff
@@ -130,3 +125,43 @@ index e1c4059..ac553bf 100644
 -- 
 2.14.1
 
+
+From e8ab90b9e91df50304e32a77325d2030c1d4aeb6 Mon Sep 17 00:00:00 2001
+From: fbrosson <fbrosson@localhost>
+Date: Tue, 1 May 2018 23:37:15 +0000
+Subject: define fake makedev, major and minor functions.
+
+
+diff --git a/libblkid/src/blkidP.h b/libblkid/src/blkidP.h
+index 22c9856..719bd83 100644
+--- a/libblkid/src/blkidP.h
++++ b/libblkid/src/blkidP.h
+@@ -341,6 +341,10 @@ UL_DEBUG_DECLARE_MASK(libblkid);
+ #define UL_DEBUG_CURRENT_MASK	UL_DEBUG_MASK(libblkid)
+ #include "debugobj.h"
+ 
++extern unsigned int major(dev_t devno);
++extern unsigned int minor(dev_t devno);
++extern dev_t makedev(unsigned int ma, unsigned int mi);
```

Comment:
```diff
++extern unsigned int major(dev_t devno);
++extern unsigned int minor(dev_t devno);
++extern dev_t makedev(unsigned int ma, unsigned int mi)
```
Would it be acceptable to move these declarations (or variants of it) to the system headers?